engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -119,6 +119,9 @@ public class DbMigrationState implements MutableMigrationState {
   private final DbProcessMessageSubscriptionMigrationState processMessageSubscriptionMigrationState;
   private final DbJobMigrationState jobMigrationState;
   private final DbSignalSubscriptionMigrationState signalSubscriptionMigrationState;
+  private final ColumnFamily<DbString, DbString> migrationsState;
+  private final DbString migratedByVersionKey = new DbString();
+  private final DbString migratedByVersionValue = new DbString();
 
```

Comment:
❓ What do you think about not adding a new CF, but using the default? Would make sense to me since it is also the meta data which I somehow would expect there.